readme/vaccination.md

```diff
@@ -0,0 +1,34 @@
+## Vaccination
+This report searches different vaccinations. It is a MSPP Report requested by the Haitian MoH to any clinic in Haiti.
+
+**Setup**
+
+It is setup by setting the following properties in [initializer](https://github.com/mekomsolutions/openmrs-module-initializer) `jsonkeyvalues`' domain configuration. 
+
+```bash
+{
+    ...
+    ...,
+        "report.vaccination.active": "true",
+        "report.ecv.active": "true",
+        "report.vaccination.prenatalVisitType": "uuid-to-prenatal-visitType",
+        "report.vaccination.vaccinations": "uuid-to-vaccinations-question-concept",
+        "report.vaccination.vaccinationSequenceNumberConcept": "uuid-to-vaccination-sequence-number-concept",
+        "report.vaccination.boostervaccinationSequenceNumberConcept": "uuid-to-vaccination-booster-number-concept",
+        "report.vaccination.vaccinationConceptsListWithSequence":"uuids-to-vaccinations:booster-or-sequence-number",
+        "report.vaccination.ecvList" : "uuid-to-ecv:booster-or-sequence-number",
+        "report.vaccination.vaccinationDate" : "uuid-to-vaccination-date-concept"
+
+}
+```
+`report.vaccination.active` activates the vaccinations report to be usable when the module is loaded (without the ECV row).
+`report.ecv.active` activates the ECV part of the vaccinations report to be usable when the module is loaded.
+`report.vaccination.prenatalVisitType` specifies the UUID of the prenatal visit type to filter on.
+`report.vaccination.vaccinations` specifies the UUID of the vaccination question concept.
+`report.vaccination.vaccinationSequenceNumberConcept`specifies UUID of the vaccination sequence number concept.
+`report.vaccination.boostervaccinationSequenceNumberConcept` specifies the UUID of the vaccination booster number concept.
```

Comment:
Rename to `boosterSequenceNumberConcept`

Reply:
Done